README.md

```diff
@@ -155,8 +155,8 @@ Then open a browser and navigate to http://localhost:3000/
 
 4.  Download and unzip the default SlashDB configuration files while in the local target directory:
 
-        wget -c https://downloads.slashdb.com/latest/default-slashdb-configs.zip
-        unzip default-slashdb-configs.zip
+        wget -c https://downloads.slashdb.com/versions/1.7.29/default-slashdb-configs_1.7.29.zip
+        unzip default-slashdb-configs_1.7.29.zip
 
```

Comment:
An earlier command `docker pull slashdb/slashdb` will be pulling the latest config. @LoipesMas do you think we could resolve the version automatically somehow or do we need the latest without version?

Reply:
I don't think there is an easy way to resolve the version automatically. If we want these commands to always get the latest version, then we need to make sure that the hosted file has correct name, i.e., without the version appended.

Reply:
Done